dbms/src/Interpreters/Context.cpp

```diff
@@ -1547,23 +1548,9 @@ ReadLimiterPtr Context::getReadLimiter() const
     return getIORateLimiter().getReadLimiter();
 }
 
-static bool isUsingPageStorageV3(const PathPool & path_pool, bool enable_ps_v3)
+static bool isPageStorageV2NotExisted(const PathPool & path_pool)
```

Comment:
Need check `PageFile_0_x` in that path or not.

also need add V3 check then changed run mode